packages/common/src/dto/contact-information.dto.ts

```diff
@@ -28,8 +29,11 @@ export class ContactInformationDTO {
 
   @IsString()
   @IsOptional()
-  @Length(0, 255, {
-    message: '255 character limit',
+  @Length(0, 3, {
```

Comment:
This field was free-text by design because extensions can be complex or have multiple steps
> dial 3, then 320

The validation described in the ticket should be disabling the field if phone number hasn't been entered

Reply:
Was this for EHPR-134?

Reply:
Yes! Ugh, I mislabled the branch.